etc/kayobe/environments/ci-multinode/networks.yml

```diff
@@ -73,7 +73,7 @@ storage_mgmt_net_name: storage_mgmt
 
 # Internal network
 internal_cidr: 192.168.37.0/24
-internal_mtu: 1450
+internal_mtu: 1392
```

Comment:
As discussed, this probably needs to be the MTU of admin_oc_interface minus 50 bytes.

Reply:
Done